src/tests/functional/plugin/conformance/subgraphs_dumper/include/utils/model.hpp

```diff
@@ -82,7 +86,8 @@ generate_model(ov::NodeVector& nodes,
                             if (cloned_node_map.count(orig_in_node_name)) {
                                 auto orig_in_node = cloned_node_map[orig_in_node_name];
                                 auto cloned_in_node_name = cloned_in_node->get_friendly_name();
-                                ov::replace_output_update_name(cloned_in_node->output(out_idx), orig_in_node->output(out_idx));
+                                // cloned_in_node is parameter or constant, it could have only one input
+                                ov::replace_output_update_name(cloned_in_node->output(0), orig_in_node->output(out_idx));
```

Comment:
Look like invalid changes... Op can have many outputs

Reply:
orig_in_node yes, cloned_in_node is parameter/constant, which was created in clone_node, it always have only 1 output

Reply:
In case if `cloned_in_node is parameter/constant` it is valid, but invalid in case we try to connect existing nodes...

Reply:
fixed